src/web/theme/modules/Tile/Tile.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ContentTile from "./ContentTile";
+import "./Tile.scss";
+
+const Tile = ({ title, children, style, contentStyle }) => {
+  const TileClass = `tile${style ? "--" + style : ""}`;
+
+  return (
+    <div className={TileClass}>
+      <ContentTile style={contentStyle} title={title}>
```

Comment:
The name of the component suggests that it is related to the `Tile` component. These are 2 very simple components, maybe you can remove complexity by not extracting `ContentTile` in his own file. Unless it is intended to be reused elsewhere obviously.

Reply:
i fixed it in the our selection #4 branch since there was changement on these files